woodwork/table_accessor.py

```diff
@@ -39,6 +39,17 @@
 ks = import_or_none('databricks.koalas')
 
 
+def _check_df_schema(func):
```

Comment:
Could/should we combine the two decorator functions into one? Or is there a reason outside of the error message to keep them as separate functions?

Not a big deal either way, just thought I'd mention as a possible option.

Reply:
The slightly different error message is the only reason they're separate.  I don't really want to make the message configurable since that would add a possible kwarg collision

Reply:
I guess it could check `self` to get that info

Reply:
@rwedge Yeah, you could do something like `class_name = repr(type(self))`.  And then use `{class_name}` in the exception message.